projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -267,8 +585,41 @@ export function checkLinkedItemInViewport(index: number) {
  * Register GET quote route.
  */
 export function registerGetQuoteRoute(shopName: string) {
-  cy.intercept({
-    method: 'GET',
-    path: `${Cypress.env('OCC_PREFIX')}/${shopName}/users/current/quotes/*`,
-  }).as(GET_QUOTE_ALIAS.substring(1)); // strip the '@'
+  trace(
```

Comment:
Please, use your `log()` method instead of `trace()` one

Reply:
done